rst_files/harrison_kreps.rst

```diff
@@ -435,8 +455,40 @@ Investors of type :math:`a` want to sell the asset in state :math:`1` while inve
 
 Here's code to solve for :math:`\bar p`, :math:`\hat p_a` and :math:`\hat p_b` using the iterative method described above
 
-.. literalinclude:: /_static/code/harrison_kreps/hk_price_optimisticbeliefs.jl
+.. code-block:: julia 
+
+    """
+    Function to Solve Optimistic Beliefs
+    """
```

Comment:
Delete docstring

Reply:
OK done. I'll do these when I catch them, but in general I think we should reserve these PRs for functional improvements.